libraries/Microsoft.Bot.Connector/Authentication/EndorsementsValidator.cs

```diff
@@ -17,18 +17,18 @@ public static class EndorsementsValidator
         /// <see cref="Schema.Activity.ChannelId"/> property is set to "webchat" and the signing party
         /// of the JWT token must have a corresponding endorsement of “Webchat”.
         /// </summary>
-        /// <param name="channelId">The ID of the channel to validate, typically extracted from the activity's
+        /// <param name="expectedEndorsement">The expected endorsement. Generally the ID of the channel to validate, typically extracted from the activity's
```

Comment:
The right way to think about this is:

- In nearly all cases, the channel ID is the required endorsement
- In some cases, the developer may elect to require an additional endorsement, but this is always additive to the channel ID endorsement
- In very rare cases, a developer will elect to remove the channel ID endorsement, but they are operating in a less secure / unsupported mode at this point.